### ISSUE TITLE
Is it good on moderate-frequency IMU?

### ISSUE THREAD

**Issue by You-zeyi** [APRIL-ZJU/OA-LICalib#2]
On your paper, it says it was experimented on 400hz and 200hz IMU data. I am currently using a 100 hz IMU from my INS. Can I still get good calibration result from using your codebase?

**Comment by arjunskumar** [comment]
@You-zeyi have you got the better results?

**Comment by pedestrain123** [comment]
@JaySlamer 您好请问您试了这个标定算法了吗？标定结果如何

**Comment by JaySlamer** [comment]
> @You-zeyi have you got the better results?

no, the result is horrible. I have already left the company and project.

**Comment by JaySlamer** [comment]
> @JaySlamer 您好请问您试了这个标定算法了吗？标定结果如何

结果很糟糕，已经被裁员，不在那个项目了

**Comment by pedestrain123** [comment]
@JaySlamer 好的感谢您的回复，您有用过opencalib的lidar2ins自动标定算法吗？基于balm的
